packages/glasscoder/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+pkgname=glasscoder
+pkgver=1.0.4
+pkgrel=1
+pkgdesc="Minimalist audio encoder for IceCast and Shoutcast streaming
+servers as well as HTTP Live Streaming (HLS)"
```

Comment:
No line-breaks in `pkgdesc`, Use single quotes.

Reply:
Also, according to https://wiki.archlinux.org/title/PKGBUILD#pkgdesc
> This is recommended to be 80 characters or less

Maybe sth like `Minimalist audio encoder for IceCast, Shoutcast and HLS`

Reply:
You're right. I've shortened that.